apps/web/pages/task/[id].tsx

```diff
@@ -68,7 +68,7 @@ const TaskDetails = () => {
 							<RichTextEditor />
 							{/* <TaskDescriptionBlock /> */}
 							<IssueCard related={false} />
-							<IssueCard related={true} />
+							{/* <IssueCard related={true} /> */}
```

Comment:
@paradoxe35 that one with `related={true}` is not implemented yet or you forgot to uncomment?

Reply:
Not yet implemented